README.md

```diff
@@ -1,19 +1,19 @@
 # Wasm Zone
```

Comment:
Do we need to keep this README intact?
For example, cosmwasm simply attaches a link to the original repository as follows.
https://github.com/line/cosmwasm/blob/main/README.md

Reply:
OK, I see.